components/Home/Home.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { QueryClient } from "react-query";
+import { dehydrate } from "react-query/hydration";
+import { Layout, InfoBox, ProductList } from "../../components";
+import { fetchPosts, fetchProducts } from "../../hooks";
+import Banner from "./Banner";
+import BigHotDig from "./BigHotDig";
+import { Content } from "./Home.styles";
+import LatestProducts from "./LatestProducts";
+import MemberList from "./MemberList";
+import Products from "./Products";
+import PolProductList from "../../components/POLProductList";
+import { useMediaQuery } from "react-responsive";
+import MobileLatest from "./MobileLatest";
+import data from "./home.json";
+export const Home = () => {
+  const isMobile = useMediaQuery({ maxWidth: 767 });
+  return (
+    <Layout>
+      <Banner />
+      <Content>
+        {isMobile ? null : <MemberList data={data.memberList} />}
```

Comment:
These statements in this return block should all be assigned to  const variables